src/deserialize.ts

```diff
@@ -1,7 +1,8 @@
-export function camelize(_: string, value: any) {
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Can we just disable this rule on the line where it's actually used, rather than on the complete file, using something like `// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment`?

Reply:
I'd have to update `.eslintrc` because currently we say that the only eslint comments that are allowed are `eslint-disable` and `eslint-enable`, so that doesn't include `eslint-disable-next-line`. But from reading https://github.com/github/codeql-variant-analysis-action/pull/739 I don't think it was intended to disallow `eslint-disable-next-line` but rather in that PR we didn't need it.